### ISSUE TITLE
Automate dm+d coding system updates

### ISSUE THREAD

**Issue by rebkwok** [opensafely-core/opencodelists#1469]
dm+d is released weekly, the releases are big, and importing takes a while.  It's a pain to do this manually, but there's an API we should be able to use to automate it:
https://isd.digital.nhs.uk/trud/users/authenticated/filters/0/api

Ideally other coding systems could be automated too, but as they're less frequent, it's not such a burden to do it manually

**Comment by rebkwok** [comment]
#1478 does the first part of this, using the API to fetch the specified release (but still with a manually run command)

**Comment by rebkwok** [comment]
#1486 does the next part, fetching the latest release

This should mean that automating it is just a matter of a cronjob that runs 
```
dokku run opencodelists python manage.py import_latest_dmd_data /storage/data/dmd/ && dokku ps:restart opencodelists
```
to import the latest release and restart the dokku app

**Comment by rebkwok** [comment]
Closed by #1501
